src/test/java/com/ays/assignment/model/dto/request/AssignmentUpdateRequestBuilder.java

```diff
@@ -0,0 +1,54 @@
+package com.ays.assignment.model.dto.request;
+
+import com.ays.common.model.AysPhoneNumber;
+import com.ays.common.model.AysPhoneNumberBuilder;
+import com.ays.common.model.TestDataBuilder;
+
+public class AssignmentUpdateRequestBuilder extends TestDataBuilder<AssignmentUpdateRequest> {
+    public AssignmentUpdateRequestBuilder() {
+        super(AssignmentUpdateRequest.class);
+    }
+
+    public AssignmentUpdateRequestBuilder withValidFields() {
+        return this
+                .withDescription("Updated Description")
+                .withFirstName("Updated First Name")
+                .withLastName("Updated Last Name")
```

Comment:
Buradaki değerleri zaten `EasyRandom` kütüphanesi dolduruyor, sadece kendi belirleyemediği durumlarda bizim değer ataması yapmamız yeterli olur. Örneğin `latitude` ve `longitude` gibi alanlarda.

Reply:
Düzelttim